Tests/TestErrors.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  TestErrors.swift
+//  RxSwiftExt
+//
+//  Created by Anton Nazarov on 25/05/2019.
+//  Copyright © 2019 RxSwift Community. All rights reserved.
+//
+
+enum TestError: Error {
```

Comment:
Instead of using an one-case enum for errors, I would suggest using a `struct` instead.

Reply:
Enum can be extended with another cases if needed. This code was copied from main repo

Reply:
If more cases are needed than go with an `Enum`, but _now_ (especially in this PR) you need only one.

Reply:
What would you gain from using a Struct here? I think it's worth just leaving as an enum because it's bound to be extended (as it spans the entire repo, not just one test)